src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -0,0 +1,696 @@
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            SomeMessage message = new SomeMessage();
+            // Make the request
+            Shelf response = await libraryServiceClient.GetShelfAsync(formattedName, message);
+            // End snippet
+        }
+
+        public void GetShelf()
+        {
+            // Snippet: GetShelf(string,SomeMessage,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            SomeMessage message = new SomeMessage();
+            // Make the request
+            Shelf response = libraryServiceClient.GetShelf(formattedName, message);
+            // End snippet
+        }
+
+        public async Task GetShelfAsync()
+        {
+            // Snippet: GetShelfAsync(string,SomeMessage,StringBuilder,CallSettings)
+            // Additional: GetShelfAsync(string,SomeMessage,StringBuilder,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            SomeMessage message = new SomeMessage();
+            StringBuilder stringBuilder = new StringBuilder();
+            // Make the request
+            Shelf response = await libraryServiceClient.GetShelfAsync(formattedName, message, stringBuilder);
+            // End snippet
+        }
+
+        public void GetShelf()
+        {
+            // Snippet: GetShelf(string,SomeMessage,StringBuilder,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            SomeMessage message = new SomeMessage();
+            StringBuilder stringBuilder = new StringBuilder();
+            // Make the request
+            Shelf response = libraryServiceClient.GetShelf(formattedName, message, stringBuilder);
+            // End snippet
+        }
+
+        public async Task ListShelvesAsync()
+        {
+            // Snippet: ListShelvesAsync(string,int?,CallSettings)
+            // Additional: ListShelvesAsync(string,int?,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+
+            // Make the request
+            IPagedAsyncEnumerable<ListShelvesResponse,Shelf> response = await libraryServiceClient.ListShelvesAsync();
```

Comment:
We might want to consider adding a line break after the `=` here, partly because docfx has such narrow docs.